prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -183,6 +183,13 @@ class FHIRTranslator(
         return newBundle
     }
 
+    /** adapt to be called from integration tests
+     *
+     */
+    fun testPruneBundleForReceiver(bundle: Bundle, receiverEndpoint: Endpoint): Bundle {
+        return pruneBundleForReceiver(bundle, receiverEndpoint)
+    }
```

Comment:
Can we remove this? I think it'll be fine to call `pruneBundleForReceiver` directly from the integration test.